src/stream/index.ts

```diff
@@ -213,4 +212,78 @@ export default class Stream extends SafeEventEmitter<StreamEventArguments> {
         this._buffer = this._buffer.subarray(count);
         this._bufferSize -= count;
     }
+
+    /**
+     * Determine the best suitable control character for a given size, along with the remainder after
+     * using that control character.
+     * @param size The size in bytes
+     */
+    private _bestControlCharacter(
```

Comment:
Lots of repetition (`number: Math.min(255, Math.floor(<> / 4) - 1)`, `remainder: Math.floor(<> / 4) - 1 > 255 ? <> - 4 * 256 : <> % 4`)
All the returns (except the first two) have the same shape
They should therefore be extracted to a function

Reply:
done in 8294489b717f189185a1442640eced765e5d7aa4

Reply:
done